src/server/enrollment-system.js

```diff
@@ -63,22 +63,163 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
-function zeroPadNumber(number, padding) {
-  return (new Array(padding + 1).join('0') + number).slice(-padding);
+/**
+ * Returns a string left-padded with '0' of the given width.
+ *
+ * Warning: If 'number' is wider than 'width' characters, only the right-most 'width'
+ * characters are returned.
+ *
+ * TODO: should this explicitly check if 'number' and 'width' are actually numbers?
+ *
+ * @param {number} number The number you want left-padded with '0' characters.
+ * @param {number} width The width of the resulting string.
+ * @returns {String}
+ *
+ * @example
```

Comment:
@example usually goes before the @param and @return blobs.  The param + returns are often the last thing in the comment block IIRC.